google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -28,7 +28,15 @@ appengine.name=Google App Engine
 appengine.environment.name.flexible=App Engine Flexible Environment
 appengine.environment.name.mvm=App Engine Managed VMs
 appengine.environment.name.standard=App Engine Standard Environment
-appengine.tools.menu.item.label=Deploy to App Engine...
+appengine.tools.menu.deploy.text=Deploy to App Engine...
+appengine.tools.menu.deploy.description=Launch App Engine Deployment Configuration Panel
+appengine.tools.menu.debug.text=Attach Stackdriver Debugger
```

Comment:
I'm not sure I follow, can you clarify please? Do you mean using the "cloudbug.*" property instead? I think keeping them grouped together like I have will make it convenient if/when we adjust them.